tests/functional/test_workspaces.py

```diff
@@ -17,23 +15,12 @@ def setUp(self):
         self.buckets = Buckets(os.getenv('KBC_TEST_API_URL'), os.getenv('KBC_TEST_TOKEN'))
         self.jobs = Jobs(os.getenv('KBC_TEST_API_URL'), os.getenv('KBC_TEST_TOKEN'))
         self.tables = Tables(os.getenv('KBC_TEST_API_URL'), os.getenv('KBC_TEST_TOKEN'))
-        self.files = Files(os.getenv('KBC_TEST_API_URL'), os.getenv('KBC_TEST_TOKEN'))
-        try:
-            file_list = self.files.list(tags=['sapi-client-python-tests'])
-            for file in file_list:
-                self.files.delete(file['id'])
-        except exceptions.HTTPError as e:
-            if e.response.status_code != 404:
-                raise
```

Comment:
Ah, I see no files are created in this test, 👍

Reply:
Yeah, that's why i split it previously, but left this bit over